scripts/upgrade-apps.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# This script automates the upgrade of the installed apps in overlays/x86/kustomization.yaml
+#
+
+set -euo pipefail
+
+KUSTOMIZATION_FILE="overlays/x86/kustomization.yaml"
+
+# Pre-fetch linuxserver.io API data to avoid multiple calls
+LSIO_API_DATA=$(curl -s "https://api.linuxserver.io/api/v1/images?include_config=false&include_deprecated=false")
+
+# Function to get the latest tag for a Docker image
+get_latest_tag() {
+    local image_name=$1
+    local current_tag=$2
+    local tags_url
+    local latest_tag
+
+    echo "Fetching tags for $image_name..." >&2
+
+    if [[ "$image_name" == "lscr.io/"* ]]; then
+        echo "Skipping lscr.io image ($image_name) as per repository policy." >&2
+        echo "$current_tag"
+        return
```

Comment:
No need to skip lscr.io, this is the linuxserver oci images registry. If image starts with this prefix use linux server tags